proto/p4/p4runtime.proto

```diff
@@ -393,25 +393,39 @@ message SetForwardingPipelineConfigRequest {
     VERIFY_AND_COMMIT = 3;
     COMMIT = 4;
   }
-  Uint128 election_id = 1;
-  repeated ForwardingPipelineConfig configs = 2;
+  uint64 device_id = 1;
+  Uint128 election_id = 2;
   Action action = 3;
+  ForwardingPipelineConfig config = 4;
+
+  // TODO(wmohsin): Remove when clients have migrated to 'device_id' and
+  // 'config' above.
+  repeated ForwardingPipelineConfig configs = 2 [deprecated = true];
```

Comment:
you cannot use the same protobuf field id twice (2), even for a deprecated field, it is causing the build to fail
apart from this, looks good to me

Reply:
Yes the field id needs to change.

Reply:
Done